### ISSUE TITLE
Clicking any input causes crash

### ISSUE THREAD

**Issue by yuraj11** [kotlin-graphics/imgui#70]
imgui v1.67-beta-00
```
Exception in thread "main" java.lang.IndexOutOfBoundsException: Index: 0, Size: 0
	at java.util.ArrayList.rangeCheck(ArrayList.java:657)
	at java.util.ArrayList.get(ArrayList.java:433)
	at imgui.imgui.imgui_internal$DefaultImpls.inputTextEx(internal.kt:2270)
	at imgui.ImGui.inputTextEx(imgui.kt:69)
	at imgui.imgui.imgui_internal$DefaultImpls.inputTextEx$default(internal.kt:2075)
	at imgui.imgui.widgets.inputKeyboard$DefaultImpls.inputText(inputKeyboard.kt:46)
	at imgui.ImGui.inputText(imgui.kt:69)
	at imgui.imgui.widgets.inputKeyboard$DefaultImpls.inputText$default(inputKeyboard.kt:42)
	at imguitest.guitest.render(guitest.kt:46)
	at com.badlogic.gdx.backends.lwjgl3.Lwjgl3Window.update(Lwjgl3Window.java:399)
	at com.badlogic.gdx.backends.lwjgl3.Lwjgl3Application.loop(Lwjgl3Application.java:137)
	at com.badlogic.gdx.backends.lwjgl3.Lwjgl3Application.<init>(Lwjgl3Application.java:111)
	at imguitest.lwjgl3.Lwjgl3Launcher.createApplication(Lwjgl3Launcher.java:14)
	at imguitest.lwjgl3.Lwjgl3Launcher.main(Lwjgl3Launcher.java:10)
```

**Comment by Sylvyrfysh** [comment]
This is fixed by 4bfa8eb . How are you getting your imgui?

**Comment by yuraj11** [comment]
I have cloned latest version from repo and your fix has fixed the problem :) 
BTW: There is another problem with inputs:
![image](https://user-images.githubusercontent.com/5477518/52646999-2a620980-2ee4-11e9-8e76-3fd6618641e0.png)

**Comment by Sylvyrfysh** [comment]
Fixed by 14180ba
